async/amqp/option.go

```diff
@@ -41,3 +41,14 @@ func Requeue(requeue bool) OptionFunc {
 		return nil
 	}
 }
+
+func Bindings(bindings ...string) OptionFunc {
+	return func(c *consumer) error {
+		if len(bindings) == 0 {
```

Comment:
the empty case should not be checked, if someone used the option he wants to add bindings so if none is provided an error should be return.

Reply:
Updated the function to return an error if the bindings are empty